src/routes/configuration/environment-variables.mdx

```diff
@@ -43,4 +57,36 @@ function MyComponent() {
 }
 ```
 
-For more information on environment variables in Vite and enabling TypeScript intellisense for environment variables, please refer to the [Vite Documentation](https://vitejs.dev/guide/env-and-mode.html#env-files).
+## Private Environment Variables
+
+These variables should only be accessed in your backend code, and so it's best to not use `VITE_` prefix for them. Instead, use `process.env` to access them. Depending on the [Nitro preset](https://nitro.unjs.io/deploy) chosen, they'll be made available automatically or they will require an external dependency such as [dotenv](https://www.npmjs.com/package/dotenv).
```

Comment:
```suggestion
Private environment variables are only accessible in your backend code.
For these variables, use `process.env` instead of the `VITE_` prefix. 
Depending on the [Nitro preset](https://nitro.unjs.io/deploy) chosen, they will be made available automatically or may require an external dependency such as [dotenv](https://www.npmjs.com/package/dotenv).
```